master/reference/felix/configuration.md

```diff
@@ -108,6 +111,8 @@ The Kubernetes API datastore driver reads its configuration from Kubernetes-prov
 | Configuration parameter | Environment variable       | Description  | Schema |
 | ------------------------|----------------------------| ------------ | ------ |
 | `KubeNodePortRanges`    | `FELIX_KUBENODEPORTRANGES` | A list of port ranges that Felix should treat as Kubernetes node ports.  Only when `kube-proxy` is configured to use IPVS mode:  Felix assumes that traffic arriving at the host one one of these ports will ultimately be forwarded instead of being terminated by a host process.  [Default: `30000:32767`] <a id="ipvs-portranges"></a>  | Comma-delimited list of `<min>:<max>` port ranges or single ports. |
+| `KubeServiceHost`       | `KUBERNETES_SERVICE_HOST`  | IP address of the kubernetes service | string |
+| `KubeServicePort`       | `KUBERNETES_SERVICE_PORT`  | Port number on which the kubernetes service is listening | int |
```

Comment:
Why do these not follow the usual convention?  (Which would be `KubeServiceHost` and `FELIX_KUBESERVICEHOST`.)  It makes me question whether these environment variables and configuration parameters will really work.

Also please capitalize Kubernetes, again.

Reply:
KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT are the env variables. Eric has confirmed these and suggested to add this in this file itself.
You can check his comment on the Jira ticket - https://tigera.atlassian.net/browse/OS-4481

Reply:
OK, I'm happy with the env variables.  But I don't think it really works to specify `KubeServiceHost` and/or `KubeServicePort` in a configuration file.  Or does it?

Reply:
Agree with @neiljerram that probably we do not need to include these parameters in our doc because it is injected automatically by kubernetes.

Reply:
I think @2ffs2nns wanted the env vars to be documented.  But I was wondering if we should put `-` (or something like that) in the "Configuration parameter" column, because I don't _think_ those configuration parameters will actually work.

(But I am not sure!)

Reply:
OK, after discussing this more with @song-jiang , I understand now that these env vars are not intended for user-level configuration, and hence we should not include them.  So I agree with reverting this part of the change.

(Apparently they are more like an internal mechanism between related Kubernetes components.)

Reply:
Okay. Removed the parameters